lib/game/client/engine/phaser-driver.js

```diff
@@ -1,7 +1,57 @@
+const {BaseDriver} = require("./base-driver");
+const {sc, ErrorManager} = require("@reldens/utils");
+const {Game, Input, Scene} = require('phaser');
+const {PhaserSceneDriver} = require("../scene/phaser-scene.driver");
 
-class PhaserDriver
+class PhaserDriver extends BaseDriver
```

Comment:
move to game/engine/phaser

Reply:
the idea is to have the engine folder with all the "base" classes and then create a single folder for each engine, in our case the default one will be phaser

Reply:
note: this way when the phaser folder is complete with all the drivers on it, we can fully extract it from the core, put it on it's own npm package and call it from the client-config as a plugin :)

Reply:
done!

Reply:
`lib/game/client/engine/phaser-driver.js`
> 
`lib/game/client/engine/phaser/phaser-driver.js`